plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/ColumnWithAliasFormatter.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.jdbc;
+
+import io.trino.spi.connector.ConnectorSession;
+
+import static com.google.common.base.Splitter.fixedLength;
+import static com.google.common.base.Strings.padStart;
+
+public class ColumnWithAliasFormatter
+{
+    public static final int DEFAULT_COLUMN_ALIAS_LENGTH = 30;
+    public static final int ORIGINAL_COLUMN_NAME_LENGTH = 24;
```

Comment:
no change requested yet:

notice that the code lives in trino-base-jdbc but the constants are for Oracle database limitations.
This is why it was requested to accept the max length limit as an argument to the method (easiest to test and bind in Guice) or the class itself (still ok to test, harder to bind in Guice).